.vscode/settings.json

```diff
@@ -1,8 +1,9 @@
-{
-    "python.linting.pylintEnabled": true,
-    "python.linting.enabled": true,
-    "files.associations": {
-        ".*rc": "ini",
-        ".env*": "ini"
-    },
+{
+    "python.linting.pylintEnabled": true,
+    "python.linting.enabled": true,
+    "files.associations": {
+        ".*rc": "ini",
+        ".env*": "ini"
+    },
+    "files.eol": "\n",
 }
```

Comment:
hihi  that file itself does not seem to have profited from the setting yet :)

Reply:
Good point!

Reply:
```
    "files.trimTrailingWhitespace": true,
    "files.insertFinalNewline": true,
    "files.trimFinalNewlines": true,
```-